src/modules/nfts/models/Nft.ts

```diff
@@ -6,12 +6,12 @@ import {
   Creator,
   MasterEditionV2Args,
 } from '@metaplex-foundation/mpl-token-metadata';
-import { Model } from '@/shared';
-import { MetadataAccount, MasterEditionAccount } from '@/programs/tokenMetadata';
-import { JsonMetadataLoader } from './JsonMetadataLoader';
-import { removeEmptyChars } from '@/utils';
-import { MasterEditionLoader } from './MasterEditionLoader';
-import { JsonMetadata } from './JsonMetadata';
+import { Model } from '../../../shared/index.js';
+import { MetadataAccount, MasterEditionAccount } from '../../../programs/tokenMetadata/index.js';
+import { JsonMetadataLoader } from './JsonMetadataLoader.js';
```

Comment:
Anything that's `index.js` doesn't need to be specified explicitly, the directory suffices.
Thus in this and all similar cases `../../../programs/tokenMetadata` would work fine as well.